src/Stormpath/Oauth/CreateGrantAuthenticationAttempt.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+
+namespace Stormpath\Oauth;
+
+
+use Stormpath\Resource\AccountStore;
+use Stormpath\Resource\Resource;
+
+class CreateGrantAuthenticationAttempt extends Resource
```

Comment:
This was renamed again to PasswordGrantAuthenticationAttempt to follow the rest of the naming convention.

@maranai you may want to do the same